inst/Upgrades.qml

```diff
@@ -0,0 +1,466 @@
+                }
+
+				ChangeRename { from:	"descriptivesBarPlotsConfidenceInterval";					to:		"descriptivesBarplotCiLevel"			}
+				ChangeRename { from:	"descriptivesBarPlotsZeroFix";								to:		"descriptivesBarplotYAxisFixedToZero"	}
+				ChangeRename { from:	"descriptivesPlotsRainCloud";								to:		"descriptivesRaincloudPlot"				}
+				ChangeRename { from:	"descriptivesPlotsRainCloudDifference";						to:		"differenceRaincloudPlot"				}
+				ChangeRename { from:	"descriptivesPlotsRainCloudDifferenceHorizontalDisplay";	to:		"differenceRaincloudPlotHorizontal"		}
+				ChangeRename { from:	"VovkSellkeMPR";											to:		"vovkSellke"							}
+				ChangeRename { from:	"hypothesis";												to:		"alternative"							}
+
+				ChangeJS
+                {
+	                    name:		"alternative"
+	                    jsFunction:	function(options)
+	                    {
+		                        switch(options["alternative"])
+		                        {
+			                            case "groupsNotEqual":	return "twoSided";
+			                            case "groupOneGreater":	return "greater";
+			                            case "groupTwoGreater":	return "less";
+		                        }
+	                    }
+                }
+
+				ChangeRename { from:	"normalityTests";											to:		"normalityTest"							}
+				ChangeRename { from:	"missingValues";											to:		"naAction"								}
+
+				ChangeJS
+                {
+	                    name:		"naAction"
+	                    jsFunction:	function(options)
+	                    {
+		                        switch(options["naAction"])
+		                        {
+			                            case "excludeAnalysisByAnalysis":	return "perDependent";
+			                            case "excludeListwise":				return "listwise";
+		                        }
+	                    }
+                }
+		}
+
+		Upgrade
+        {
+	            functionName: "TTestOneSample";	fromVersion: "0.16.3";		toVersion: "0.16.4";
+
+				ChangeRename { from:	"variables";									to:		"dependent"										}
+				ChangeRename { from:	"students";										to:		"student"										}
+				ChangeRename { from:	"mannWhitneyU";									to:		"wilcoxon"										}
+				ChangeRename { from:	"stddev";										to:		"sd"											}
+				ChangeRename { from:	"meanDiffConfidenceIntervalCheckbox";			to:		"meanDifferenceCi"								}
+				ChangeRename { from:	"meanDiffConfidenceIntervalPercent";			to:		"meanDifferenceCiLevel"							}
+				ChangeRename { from:	"effSizeConfidenceIntervalCheckbox";			to:		"effectSizeCi"									}
+				ChangeRename { from:	"effSizeConfidenceIntervalPercent";				to:		"effectSizeCiLevel"								}
+				ChangeRename { from:	"descriptivesPlots";							to:		"descriptivesPlot"								}
+				ChangeRename { from:	"descriptivesPlotsConfidenceInterval";			to:		"descriptivesPlotCi"							}
+				ChangeRename { from:	"descriptivesBarPlots";							to:		"descriptivesBarplot"							}
```

Comment:
so in theory the bar plot options do not need to be here as the bar plots were not yet implemented in 0.16.3, so the files saved with 0.16.3 will not contain these values in the first place.

I don't think we will merge this before the release though, so we can keep it in here. What does need to happen then is to
1. bump the module version to 0.16.4 in a separate PR so that JASP 0.16.4 actually contains the module with version 0.16.4
2. bump the module versions in this PR to `fromVersion: 0.16.4` and `toVersion: 0.16.5` (or 0.17?)

Reply:
The module version is already 0.16.4 now.